bobpay/public/index.html

```diff
@@ -225,6 +225,14 @@ <h2 class="section-heading">Install BobPay</h2>
                     </p>
                     <input id="alipay" type="checkbox" value="">AliPay</label>
                     <p id="bobpayerror" class="error-message"></p>
+                    <br>
+                    <br>
+                    <h2 class="section-heading">Setup FIDO / WebAuthn</h2>
```

Comment:
Let's put the WebAuthn code into a `<div id="webauthn">` and show it only if Web Authentication is enabled.
```html
<div id="webauthn" style="display: none;">
  ...
</div>
<script>
  if (window.WebAuthnTransaction) {
    document.getElementById('webauthn').style.display = 'block';
  }
</script>
```